html-aam/index.html

```diff
@@ -16816,7 +16816,43 @@ <h4>`iframe` Element Accessible Name Computation</h4>
           </p>
         </section>
         <section>
-          <h4>Section and Grouping Element Accessible Name Computation</h4>
+          <h4>`dialog` Element Accessible Name Computation</h4>
```

Comment:
Just an editorial question for my understanding, here and below:
Why do we need this information here? Shouldn't the mapping to the ARIA role mean that we rely on the name computation for that role, which specifies NameFrom: author, heading and thus covers the heading requirement here? Is there something that doesn't cover?